src/Hill.jl

```diff
@@ -66,3 +66,31 @@ function getDDEparams(p, concentrations)
     end
     return effects
 end
+
+function ParamForBliss(p)
+    """ To calculate Bliss independence drug effect
+    we assume delays are constant, death rates are additive,
+    and will keep the alpha and beta intact."""
+    par = zeros(3,8)
+    par[1,:] = p[1,:] # alpha stays the same
+    par[2,:] = p[2,:] # beta stays the same
+    par[3,:] = p[5,:] + p[6,:] # additivity assumption for death rates
+    return par
+end
+
+function BlissCombination(p1::Matrix{Float64}, p2::Matrix{Float64})
+    """ A function to calculate Bliss independence for drug combination assuming
+    the two drugs hit different pathways and they effect completely independently. """
+
+    param1 = ParamForBliss(p1)
+    param2 = ParamForBliss(p2)
+    Effect = zeros(8,8,3)
+    for i in 1:3
+        for j in 1:8
+            for k in 1:8
+                Effect[j,k,i] = param1[i,j] + param2[i,k] - param1[i,j] * param2[i,k]
```

Comment:
Indentation doesn't seem to line up for these loops.

Could get rid of the i loop and vectorize this:

`Effect[j,k,:] .= param1[:,j] .+ param2[:,k] .- param1[:,j] .* param2[:,k]`